src/cpp/library/cmath

```diff
@@ -48,7 +48,7 @@ double sqrt(double n) {
 
 unsigned sqrt(unsigned x) {
 	__ESBMC_assert(x >= 0, "The number must be greater or equal than 0");
-	return nondet_unsigned();
```

Comment:
There is no declaration of `nondet_unsigned()` in `clang_c_language.cpp` , whether to use `nondet_int()` instead. Its behavior is the same.

Reply:
You can declare:

unsigned nondet_unsigned();

Reply:
May I ask what is the difference between `unsigned` and `unsigned int`? In my tests, they seemed to perform the same.

Reply:
>May I ask what is the difference between unsigned and unsigned int?

No difference in this case. 
But `nondet_int()` shouldn't be used here.

Reply:
I made a mistake. I meant to use `nondet_uint`, but forgot 'u'.